src/BenchmarkDotNet/Configs/ImmutableConfigBuilder.cs

```diff
@@ -88,18 +88,22 @@ private static ImmutableHashSet<IDiagnoser> GetDiagnosers(IEnumerable<IDiagnoser
             return builder.ToImmutable();
         }
 
-        private static ImmutableArray<IExporter> GetExporters(IEnumerable<IExporter> exporters, ImmutableHashSet<IDiagnoser> uniqueDiagnosers)
+        private static ImmutableArray<IExporter> GetExporters(IEnumerable<IExporter> exporters,
+            ImmutableHashSet<IDiagnoser> uniqueDiagnosers,
+            IEnumerable<Loggers.ILogger> loggers = null)
         {
-            var result = new List<IExporter>();
+
+            var mergeDictionary = new Dictionary<System.Type, IExporter>();
 
             foreach (var exporter in exporters)
-                if (!result.Contains(exporter))
-                    result.Add(exporter);
+                mergeDictionary[exporter.GetType()] = exporter;
```

Comment:
If `exporters` contains two exporters of the same type, the first one will be overwritten with the second one without a warning.

Reply:
Before correcting, I await your decision your decision as to whether it is better to use the Analyzer.

Reply:
I have a idea. We can create a LoggerAnalyser thath implement ILogger and IAnalyser. LoggerAnalyser collect all message have logKind equal Error or Warning. This way we don't need to add any new property to IConfig.

Reply:
I don't think it's a good idea to aggregate ALL the errors/warnings under the summary table. Let's say we can't build a specific benchmark. In this case, we dump the full error message into the log. Howerver, at the end of the log, we want to just print a short message "We can't build a benchmark"  without the full build error message (which would be too overwhelmed).

Reply:
So instead of collecting warnings and errors we could count them. If they are greater than zero, we show warning.

Reply:
I think it is better to show a warning the overwrite. When an unexpected result occurs, there are more indications to find the source of the problem.